api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml

```diff
@@ -22,7 +22,7 @@ parameter_rules:
     use_template: max_tokens
     default: 1024
     min: 2
-    max: 1024
+    max: 2048
```

Comment:
hi, looks like this is a irrelevant change

Reply:
oh no, I will change it.